analysis-environments.html

```diff
@@ -0,0 +1,202 @@
+                </td>
+              {% assign count = count | plus: 1 %}
+              {% endif %}
+           {% endfor %}
+           </tr>
+           <table>
+        </div>
+
+        <div class="row column text-center">
+            <h3>Using tools to explore IDR</h2>
+        </div>
+        <div class="row align-right">
+            <table class="table">
+            {% assign count = 0 %}
+            <tr>
+            {% for item in items %}
+              {% if item.name contains "Tools" %}
+              {% assign links=item.breadcrumb %}
+              {% if count == 3 %}
+                {% assign count = 0 %}
+                </tr>
+                <tr>
+              {% endif %}
+                <td class="table-cell">
+                <table class="table">
+                <tr>
+                <td class="table-cell"><a href="{{ item.url }}" target="_blank"><img src="{{ item.image }}" alt="{{ item.name }}"/></a></td>
+                </tr>
+                <tr>
+                <td class="table-cell">{{ item.description }}</td>
+                </tr>
+                <tr>
+                <td class="table-cell">
+                {% for l in links %}
+                  <a href="{{ l.url }}" target="_blank"><img src="{{ l.image }}" alt="{{ l.name }}"/></a>
+                {% endfor %}
+                </td>
+                </tr>
+                </table>
+                </td>
+              {% assign count = count | plus: 1 %}
+              {% endif %}
+           {% endfor %}
+           </tr>
+           <table>
+        </div>
+        <!-- end Notebooks section -->
+
+
+
+       <hr class="whitespace">
+        <div class="row column text-center">
+            <h2>Developing your own</h2>
+        </div>
+        <div class="row">
+            <div class="small-10 small-centered medium-10 medium-centered columns">
+                <div class="row horizontal">
+                    <div>
+                      <h1 class="text-center">GitHub login</h1>
+                      <p>
+                        If you are interested in developing your own data analysis notebooks on our analysis platform please contact us with your GitHub username.
+                        This will give you more computational resources including scratch storage space than is available through the public access.
+                      </p>
+                      <p>
+                        We will do our best to maintain this scratch space across upgrades, though in the event of unforeseen issues we may need to delete all scratch space, and you should consider it as suitable for temporary storage only.
+                        Your scratch space may be deleted without warning if you have not logged in for over a month.
+                      </p>
+                      <div class="row column text-center">
+                        <a href="https://login.binder.bioimagearchive.org/" class="large button sites-button">Login with GitHub</a>
```

Comment:
This seems to work, so do I still need to contact as indicated above or can I just start developing my notebooks here ?

Reply:
Text adjusted